config/jobs/periodic/containerd/test-containerd-powervs-periodic.yaml

```diff
@@ -0,0 +1,64 @@
+periodics:
+  - name: periodic-containerd-test-powerVS
+    cluster: k8s-ppc64le-cluster
+    decorate: true
+    interval: 24h
+    spec:
+      nodeSelector:
+        dedicated: runtimesTeam
+      tolerations:
+      - key: dedicated
+        operator: "Equal"
+        value: runtimesTeam
+        effect: "NoSchedule"
+      volumes:
+      - name: atos-ssh-volume
+        secret:
+          secretName: atos-powervs-ssh-key
+          defaultMode: 256
+      containers:
+        - image: quay.io/powercloud/all-in-one:0.3
+          volumeMounts:
+          - name: atos-ssh-volume
+            readOnly: true
+            mountPath: "/etc/ssh-volume"
+          command:
+            - /bin/bash
+            - -c
+            - |
+              set -o errexit
+              set -o nounset
+              set -o pipefail
+              set -o xtrace
+              export KUBE_TIMEOUT='--timeout=600s'
+              export KUBE_KEEP_VERBOSE_TEST_OUTPUT=y
+              export LOG_LEVEL=4
+
+              # Connect to IBMCloud
+              echo "" | ibmcloud login
+              ibmcloud target -r eu-gb -g runtimes-dev-resource-group
+              ibmcloud pi service-target crn:v1:bluemix:public:power-iaas:lon06:a/65b64c1f1c29460e8c2e4bbfbd893c2c:5e18d40d-c743-4274-8973-6cbfaf6bf12b::
+
+              wget https://raw.githubusercontent.com/ppc64le-cloud/docker-ce-build/main/test-containerd/instanciate_powervs_vm.sh
```

Comment:
From the build log of job https://prow.ppc64le-cloud.org/view/s3/prow-logs/logs/periodic-containerd-test-powerVS/1504756840649461760 I see tests for the below configuration does not seem to have run:

`--runc runc --runtime "io.containerd.runc.v2"`

The below process seems to be abruptly **missing/failing** but still, the prow job result is seen as PASSED:
```
bash -x ./instanciate_powervs_vm.sh --output ${ARTIFACTS} --name rdr-runtimes-dev-etienne_guesnet-crun_v2 --runc crun --runtime "io.containerd.runc.v2" --key atos-key --network 'public-192_168_141_8-29-VLAN_2093' &
```
Looks like `instanciate_powervs_vm.sh` needs improvisation to tell prow job if there is any failure while VM deployment.

Also, I see all the junit xml files from four different configurations being placed under the same ARTIFACTS folder

(I understand that they have different naming conventions) hence the Junit tab in [UI](https://prow.ppc64le-cloud.org/view/s3/prow-logs/logs/periodic-containerd-test-powerVS/1504756840649461760) does not distinguish the tests with respect to configuration under which it is run.

@EGuesnet you may explore how to make the UI distinguish based on the configuration in which it was run. 

@alunsin ^^